_dashboards/im-dashboards/datastream.md

```diff
@@ -61,17 +61,17 @@ To delete a data stream, perform the following steps:
 
 ## Rolling over a data stream
 
-To perform a rollover operation on a data stream, perform the following steps:
+To perform a roll over operation on a data stream, perform the following steps:
```

Comment:
Should we explain why someone would want to perform a roll over on a data stream?

Reply:
This is explained in the ISM/data-stream/API documentation as to the “why” you would do it. As this is a front-end for an existing feature I wouldn’t think so. Also, I have linked to that content in this page.

Reply:
This should be rollover because it's used as an adjective
```suggestion
To perform a rollover operation on a data stream, perform the following steps:
```